Yoast/wordpress-seo
# Make woocommerce integration

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Install WooCommerce and setup a shop page. (this should be done automatically)
* Visit the shop page and check if you have the right title and description.

## UI changes

## Documentation

## Quality assurance


Fixes #
